app.js

```diff
@@ -45,24 +43,50 @@ arrayOfFiles.forEach(function(file) {
   discoveryDocs.push(path.join('./data/breaches/', file));
 });
 
-// let discoveryParams; // discoveryParams will be set after Discovery is validated and setup.
+// setup Discovery by finding/creating environment and finding/creating collection
+var discoverySetupParams = { default_name: DEFAULT_COLLECTION_NAME };
 const discoverySetup = new WatsonDiscoverySetup(discovery);
-const partialDocs = discoveryDocs.slice(0, 15);
-const discoverySetupParams = { default_name: DEFAULT_NAME, documents: partialDocs };
 discoverySetup.setupDiscovery(discoverySetupParams, (err, data) => {
   if (err) {
     handleSetupError(err);
   } else {
-    console.log('Discovery is ready!' + data);
-    // discoveryParams = data;
+    console.log('Discovery is ready!');
+
+    // now load data into discovery service collection
+    var collectionParams = data;
+
+    // set collection creds - at this point the collectionParams
+    // will point to the actual credentials, whether the user
+    // entered them in .env for an existing collection, or if
+    // we had to create them from scratch.
+    environment_id = collectionParams.environment_id;
+    collection_id = collectionParams.collection_id;
+    
+    collectionParams.documents = discoveryDocs;
+    console.log('Begin loading ' + discoveryDocs.length + ' json files into discovery. Please be patient as this can take several minutes.');
+    loadCollectionFiles(collectionParams);
   }
 });
 
+// load json files into discovery collection
```

Comment:
nit -- when we have time we need to use proper /** docs (like javadocs) for javascript functions